test/Python/python_runtime.swift

```diff
@@ -309,6 +318,11 @@ PythonRuntimeTestSuite.testWithLeakChecking("PythonConvertible") {
   expectEqual(five, UInt64(5).pythonObject)
   expectEqual(five, Float(5).pythonObject)
   expectEqual(five, Double(5).pythonObject)
+
+  expectEqual(intArray, Array([2, 3]).pythonObject)
```

Comment:
Array literal `[2, 3]` already resolves to an `Array`.
```suggestion
  expectEqual(intArray, [2, 3].pythonObject)
```